### ISSUE TITLE
Cleanup `package.json`/JS scripts

### ISSUE THREAD

**Issue by jhonatan-lopes** [MozillaFoundation/foundation.mozilla.org#11767]
The `scripts` section of our `package.json` could use some deep cleaning. Some commands are duplicated in our invoke `tasks.py` (such as `server`, `docker:up` and `docker:down`).

Ideally, we should remove any tasks that are already present in `tasks.py` and migrate the ones that can be migrated to keep everything in a single place. IMO, it is better to use `invoke`, as the tool was built specifically for this.

We should also make sure that dependencies needed here, such as `playwright`, are installed during the Docker build phase, not after, to keep consistency between the dev environment and prod/staging environments.

Another task here is to also migrate the `copy-db.js` command to `invoke`, as it has lots of duplicates with commands already defined in `tasks.py`

To sum up, the tasks here should be:

* Remove any duplicated scripts that have equivalents on `tasks.py`
* Migrate scripts that do not have equivalents on `tasks.py`
* Install `playwright` and other dependencies in the dev Docker build
* Migrate the `copy-db.js` command to invoke

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/TP1-239)
